index.html

```diff
@@ -2,9 +2,141 @@
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <a href="#contact" class="link-pink">Let's work together</a>
+        </div>
+        <span class="squiggle-stripe"><img src="assets/squiggle_line_stripe.svg" alt="squiggle stripe"/></span>
+    </div>
+</header>
 
+<main id="case-studies">
+    <div class="outer-container outer-introduction">
+        <div class="inner-container inner-introduction">
+            <div class="introduction-text">
+                <h2>Rekupe is a digital agency in Los Angeles.</h2>
+                <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
+                <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+                <p>Have a question? <a href="#contact">Click here</a> to contact us!</p>
+            </div>
+        </div>
+    </div>
+    <section class="outer-container outer-work">
+        <div class="inner-container rolling-stone">
+            <div class="work-text-wrapper">
+                <h3>Rolling Stone</h3>
+                <div class="squiggle-wrapper"><img src="assets/squiggle_line.svg" alt="squiggle" /></div>
+                <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+                <a href="case-studies-page.html">View project</a>
+            </div>
+            <span class="image-wrapper"><img src="assets/work-image-01.jpeg" alt="work-image-01.jpeg"/></span>
+        </div>
+        <div class="inner-container variety">
+            <div class="work-text-wrapper">
+                <h3>Variety</h3>
+                <div class="squiggle-wrapper"><img src="assets/squiggle_line.svg" alt="squiggle" /></div>
+                <p>Variety is an industry leader for breaking entertainment news, reviews and box office results with
+                    archive of content dating back to 1905.</p>
+                <a href="case-studies-page.html">View project</a>
+            </div>
+            <span class="image-wrapper"><img src="assets/work-image-02.jpeg" alt="work-image-01.jpeg"/></span>
+        </div>
+        <div class="inner-container worth">
+            <div class="work-text-wrapper">
+                <h3>Worth</h3>
+                <div class="squiggle-wrapper"><img src="assets/squiggle_line.svg" alt="squiggle" /></div>
+                <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
+                <a href="case-studies-page.html">View project</a>
+            </div>
+            <span class="image-wrapper"><img src="assets/work-image-03.jpeg" alt="work-image-01.jpeg"/></span>
+        </div>
+    </section>
+    <section class="outer-container service-info-blocks" id="services">
+        <div class="inner-container inner-services">
+            <div class="info-block">
```

Comment:
Hier had je, net zoals in de feedback hierboven, beter het article element kunnen gebruiken. Hetzelfde geld voor de divjes op regel 78, 85, 92, 100 en 107. Het gebruik van semantische elementen is belangrijk, anders zouden er alleen maar divjes komen te staan en dat maakt het erg onoverzichtelijk!